FirebaseVertexAI/Sample/GenerativeAITextSample/Screens/SummarizeScreen.swift

```diff
@@ -61,6 +64,15 @@ struct SummarizeScreen: View {
   }
 
   private func onSummarizeTapped() {
+    // MARK: Close Keyboard
+
+    UIApplication.shared.sendAction(
+      #selector(UIResponder.resignFirstResponder),
+      to: nil,
+      from: nil,
+      for: nil
+    )
```

Comment:
With the above [`focused(_:equals:)`](https://developer.apple.com/documentation/swiftui/view/focused(_:equals:)), I _think_ you could set `focusedField = nil` here to close the keyboard instead.

Reply:
Yes, I did try ```focusedField = nil``` before adding this line of code.
However, it is not working on my side 🤔

![image](https://github.com/firebase/firebase-ios-sdk/assets/54872601/ad70d48b-a047-4a35-9dba-d5e15b76bcf5)